lib/db.js

```diff
@@ -54,11 +54,20 @@ class LimitDBRedis extends EventEmitter {
       redisOptions
     };
 
+    this.pingConfig = {
+      enabled: config.ping?.enabled || false,
+      interval: config.ping?.interval || 1000,
+      maxFailedAttempts: config.ping?.maxFailedAttempts || 5,
+      reconnectIfFailed: config.ping?.reconnectIfFailed || false,
+      maxFailedAttemptsToRetryReconnect: 10
```

Comment:
I'm curious. Shouldn't it be something like this?
```suggestion
      maxFailedAttemptsToRetryReconnect: config.ping?.maxFailedAttemptsToRetryReconnect || 10
```

Reply:
OMG, I didn't realize I pushed it like that.